src/schema/partner_show.js

```diff
@@ -25,6 +24,7 @@ import {
   GraphQLBoolean,
 } from "graphql"
 import { allViaLoader } from "../lib/all"
+import { totalViaLoader } from "../lib/loaders/legacy/total"
```

Comment:
Oh, yea I will eventually clean up the legacy `total`, and move the helper out of `legacy/` directory (which won't exist anymore).